elements/rhodium.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: Rhodium
+---
+
+Symbol : Rh
+Atomic Number : 45
+
+Rhodium makes up only 0.00002 ppm of the earth's crust.
+
+Fun Facts:
+- ~87% of all rhodium produced is used for catalytic converters.
+- The price of an ounce of Rhodium (~$19,000) is over 10x the price of an ounce of gold (~$1,800).
```

Comment:
<img width="812" alt="Screenshot 2024-08-05 at 5 34 25 PM" src="https://github.com/user-attachments/assets/d3afcf02-e46a-4b81-81b3-8956d3636aa8">

There is a formatting issue in this line, because of the `~` symbols, in markdown this creates a strikethrough ~like this~

You can fix this by escaping the ~ 

```suggestion
- The price of an ounce of Rhodium (\~$19,000) is over 10x the price of an ounce of gold (\~$1,800).
```